apps/avifenc.c

```diff
@@ -1561,6 +1564,15 @@ int main(int argc, char * argv[])
         goto cleanup;
     }
 
+    if (noOverwrite){
```

Comment:
Missing space before {
(or you can just run "clang-format -style=file -i apps/avifenc.c")